src/components/ComponentConfiguration.stories.tsx

```diff
@@ -7,7 +7,10 @@ import {Meta, StoryFn, StoryObj} from '@storybook/react';
 import {expect, fireEvent, fn, userEvent, waitFor, within} from '@storybook/test';
 import React from 'react';
 
-import {ReferentielijstenServiceOption} from '@/components/builder/values/referentielijsten/service';
+import {
+  ReferentielijstenServiceOption,
+  ReferentielijstenTabelItem,
+} from '@/components/builder/values/referentielijsten/types';
```

Comment:
```suggestion
import {
  ReferentielijstenServiceOption,
  ReferentielijstenTabelItem,
} from './builder/values/referentielijsten/types';
```
I generally prefer the explicit `@/` imports, but this file already has a bunch of imports so let's keep a consistent style within this file